docs-src/cloud/temporal-platform-limits-sheet.md

```diff
@@ -0,0 +1,81 @@
+---
+slug: temporal-platform-limits-sheet
+title: What are the Temporal Platform limits?
+sidebar_label: Upgrade policy
+description: Basic upgrade policy for customers of Temporal Cloud
+tags:
+  - error
+  - warn
+  - limits
+---
+
```

Comment:
Please add an introductory paragraph here that sets expectations and adds more context?

For example, "This section describes many of the default settings and limits that apply to Workflow attributes, executions, yada " -something to that effect.

Good job for ChatGPT... ;)